common.gypi

```diff
@@ -34,7 +34,7 @@
 
     # Reset this number to 0 on major V8 upgrades.
     # Increment by one for each non-official patch applied to deps/v8.
-    'v8_embedder_string': '-node.37',
+    'v8_embedder_string': '-node.38',
```

Comment:
This line had a conflict when cherrypicking, but I think this just needs an increment, right?